cgi/nutrition.pl

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/perl -w
```

Comment:
Where is `cgi/nutrition.pl` called?

Reply:
It's to be used in things like robotoff or later in the product edit form, it's not called yet.